graphql_compiler/tests/test_data_tools/integration_data/create_redisgraph_integration.cypher

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+# This data is almost identical to the neo4j integration test data except for the first line, which doesn't appear in
```

Comment:
Add a reference to the Github issue you opened about this, so we know when we can remove this black magic code.